.github/pull_request_assignment.yml

```diff
@@ -76,6 +78,20 @@
       - raych1
       - xiaoxuqi-ms
 
+- rule:
+    paths: "specification/sql/**"
+    reviewers:
+      - raych1
+      - xiaoxuqi-ms
+    branches:
+      - dev-sql-
+
+- rule:
+    paths: "specification/app/**"
+    reviewers:
+      - msyyc
+    branches:
+      - Microsoft.App-
```

Comment:
If adding branches definition here, then main branch will be still applied or added?

Reply:
These branches will be combined with default branches.